test/e2e/kubevirt.go

```diff
@@ -1775,70 +1872,81 @@ runcmd:
 			}
 		},
 			func(td testData) string {
-				role := "secondary"
+				role := udnv1.NetworkRoleSecondary
 				if td.role != "" {
 					role = td.role
 				}
-				return fmt.Sprintf("after %s of %s with %s/%s", td.test.description, td.resource.description, role, td.topology)
+				ingress := "snat"
+				if td.ingress != "" {
+					ingress = td.ingress
+				}
+				return fmt.Sprintf("after %s of %s with %s/%s with %q ingress", td.test.description, td.resource.description, role, td.topology, ingress)
 			},
 			Entry(nil, testData{
 				resource: virtualMachine,
 				test:     restart,
-				topology: "localnet",
+				topology: udnv1.NetworkTopologyLocalnet,
 			}),
 			Entry(nil, testData{
 				resource: virtualMachine,
 				test:     restart,
-				topology: "layer2",
+				topology: udnv1.NetworkTopologyLayer2,
 			}),
 			Entry(nil, testData{
 				resource: virtualMachineWithUDN,
 				test:     restart,
-				topology: "layer2",
-				role:     "primary",
+				topology: udnv1.NetworkTopologyLayer2,
+				role:     udnv1.NetworkRolePrimary,
 			}),
 			Entry(nil, testData{
 				resource: virtualMachine,
 				test:     liveMigrate,
-				topology: "localnet",
+				topology: udnv1.NetworkTopologyLocalnet,
 			}),
 			Entry(nil, testData{
 				resource: virtualMachine,
 				test:     liveMigrate,
-				topology: "layer2",
+				topology: udnv1.NetworkTopologyLayer2,
+			}),
+			Entry(nil, testData{
+				resource: virtualMachineWithUDN,
+				test:     liveMigrate,
+				topology: udnv1.NetworkTopologyLayer2,
+				role:     udnv1.NetworkRolePrimary,
 			}),
 			Entry(nil, testData{
 				resource: virtualMachineWithUDN,
 				test:     liveMigrate,
-				topology: "layer2",
-				role:     "primary",
+				topology: udnv1.NetworkTopologyLayer2,
+				role:     udnv1.NetworkRolePrimary,
+				ingress:  "routed",
```

Comment:
What happens in the lane where BGP is disabled?

Reply:
It get skip for non IC
```
	if td.role == udnv1.NetworkRolePrimary && !isInterconnectEnabled() {
				const upstreamIssue = "https://github.com/ovn-org/ovn-kubernetes/issues/4528"
				e2eskipper.Skipf(
					"The egress check of tests are known to fail on non-IC deployments. Upstream issue: %s", upstreamIssue,
				)
			}

```